fava/application.py

```diff
@@ -214,10 +214,20 @@ def should_collapse_account(account_name):
 
         return account_name in collapse_accounts
 
+    def sidebar_queries():
+        show = app.config.user.getint('sidebar-show-queries')
+        max_length = 20
+        if show > 0:
+            truncate = lambda x: x if len(x) <= max_length else x[:(max_length - 1)].rsplit(' ', 1)[0] + '\u2026'
```

Comment:
There is a way to do this in CSS, with `text-overflow: ellipsis`, but I couldn't get that to work...